manpages/awesome.1.md

```diff
@@ -0,0 +1,46 @@
+awesome(1)
+==========
+
+NAME
+----
+
+awesome - Lua controller for Way-Cooler
```

Comment:
This wording is a bit ambiguous. Here's a clearer definition about what `awesome` does.

```suggestion
awesome - a clone of AwesomeWM as a Wayland client.
```